packages/tools/devtools/devtools-core/src/Features.ts

```diff
@@ -56,7 +56,15 @@ export interface ContainerDevtoolsFeatureFlags {
 	 */
 	containerDataVisualization?: boolean;
 	/**
-	 * Indicates that this container was registered as a data object via registerDataObject().
+	 * Indicates that the Container Devtools supports connecting the container.
 	 */
-	dataObjects?: boolean;
+	canConnect?: boolean;
+	/**
+	 * Indicates that the Container Devtools supports disconnecting the container.
+	 */
+	canDisconnect?: boolean;
+	/**
+	 * Indicates that the Container Devtools supports closing the container.
+	 */
+	canClose?: boolean;
```

Comment:
I like the direction, but I think we can probably unify these into a single property? Something like `canModifyContainerState` or something? We should be sure to document what all that means though (connect, disconnect, and close).